From 3b31b9444db3ce40d50f52c06cd7eff6a9fa19b3 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Fri, 2 Dec 2016 10:02:53 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Use ext4 for ext2 file systems

also disable F2FS_FS_POSIX_ACL
save about 40KB size

Change-Id: I2a134966ece2a3f74014c059d2dd5bbccfe52b61
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 724df0931cb7..368121d2412c 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -659,11 +659,10 @@ CONFIG_ROCKCHIP_EFUSE=y
 CONFIG_RK_NAND=y
 CONFIG_RK_HEADSET=y
 # CONFIG_ACPI is not set
-CONFIG_EXT2_FS=y
-CONFIG_EXT2_FS_XATTR=y
-CONFIG_EXT3_FS=y
+CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_SECURITY=y
 CONFIG_F2FS_FS=y
+# CONFIG_F2FS_FS_POSIX_ACL is not set
 CONFIG_F2FS_FS_SECURITY=y
 CONFIG_QUOTA=y
 CONFIG_FUSE_FS=y
-- 
2.35.3

